go-controller/pkg/util/node_annotations.go

```diff
@@ -312,6 +315,39 @@ func NodeChassisIDAnnotationChanged(oldNode, newNode *kapi.Node) bool {
 	return oldNode.Annotations[ovnNodeChassisID] != newNode.Annotations[ovnNodeChassisID]
 }
 
+type ManagementPortDetails struct {
+	PfId   int `json:"PfId"`
+	FuncId int `json:"FuncId"`
+}
+
+func SetNodeManagementPortAnnotation(nodeAnnotator kube.Annotator, PfId int, FuncId int) error {
+	mgmtPortDetails := ManagementPortDetails{
+		PfId:   PfId,
+		FuncId: FuncId,
+	}
+	bytes, err := json.Marshal(mgmtPortDetails)
+	if err != nil {
+		return fmt.Errorf("failed marshaling")
+	}
+	return nodeAnnotator.Set(ovnNodeManagementPort, string(bytes))
+}
+
+// ParseNodeManagementPort returns the parsed host addresses living on a node
+func ParseNodeManagementPortAnnotation(node *kapi.Node) (int, int, error) {
+	mgmtPortAnnotation, ok := node.Annotations[ovnNodeManagementPort]
+	if !ok {
+		return -1, 0, newAnnotationNotSetError("%s annotation not found for node %q", ovnNodeManagementPort, node.Name)
```

Comment:
`return -1, -1, ...`
Sorry, I should be more specific next time :)

Reply:
done.